code/grant/pokedex/pokedex/management/commands/load_pokemon.py

```diff
@@ -0,0 +1,1953 @@
+#             "types": [
+#                 "dragon"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/dragonair"
+#         },
+#         {
+#             "number": 149,
+#             "name": "dragonite",
+#             "height": 22,
+#             "weight": 2100,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/149.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/149.png",
+#             "types": [
+#                 "flying",
+#                 "dragon"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/dragonite"
+#         },
+#         {
+#             "number": 150,
+#             "name": "mewtwo",
+#             "height": 20,
+#             "weight": 1220,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/150.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/150.png",
+#             "types": [
+#                 "psychic"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/mewtwo"
+#         },
+#         {
+#             "number": 151,
+#             "name": "mew",
+#             "height": 4,
+#             "weight": 40,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/151.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/151.png",
+#             "types": [
+#                 "psychic"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/mew"
+#         }
+#     ]
+# }
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        Pokemon.objects.all().delete()
+        PokemonType.objects.all().delete()
+
+        f = open('pokemon.json')
+
+        # with open('pokemon.json', 'w') as poke_balls:
+
+        #     json.dump(f, poke_balls)
+
+        print(f)
+
+        poke_balls = json.load(f)
+
+        print(poke_balls)
+
+
+
+        for pokemon in poke_balls:
+
+            name = pokemon.get('name'),
+            number = int(pokemon.get('number')),
+            height = float(pokemon.get('height')),
+            weight = float(pokemon.get('weight')),
+            image_front = pokemon.get('image_front'),
+            image_back = pokemon.get('image_back'),
+            types = pokemon.get('types'),        
+        
+            pokemon = Pokemon.objects.create(
+
+                number = number,
+                name = name,
+                height = height,
+                weight = weight,
+                image_front = image_front,
+                image_back = image_back,
+                types = types,
+            
+            )
+
+            poke_types, created = PokemonType.objects.get_or_create(name=types)
```

Comment:
`types` is a list of strings. The code will have to loop through this list and `get_or_create` will need to be called for each. Then each gotten or created `PokemonType` objects will be added to the list of types for the current pokemon or the current pokemon will need to be added to each type.